applications/callflow/test/cf_ring_group_test.erl

```diff
@@ -0,0 +1,21 @@
+%%%-------------------------------------------------------------------
+%%% @copyright (C) 2015, 2600Hz INC
+%%% @doc
+%%%
+%%% @end
+%%% @contributors
+%%%     SIPLABS LLC (Ilya Ashchepkov)
+%%%-------------------------------------------------------------------
+-module(cf_ring_group_test).
+
+-include_lib("eunit/include/eunit.hrl").
+
+weighted_random_sort_test() ->
+    EndpointsInt = [
+                    {1, <<"1">>}
+                    ,{2, <<"2">>}
+                    ,{3, <<"3">>}
+                   ],
+    Endpoints = cf_ring_group:weighted_random_sort(EndpointsInt),
```

Comment:
Please test multiple endpoints at a weight:

```
EndpointsInt = [{1, <<"1a">>}
                ,{1, <<"1b">>}
                ,{2, <<"2">>}
                ,{3, <<"3">>}
               ],
```

This should return a list of `[<<"1X">>, <<"1Y">>, <<"2">>, <<"3">>]` where `X` and `Y` are either `a` or `b`. Does that make sense? Make sure the endpoints are actually sorted by weight, then randomly sorted within their weight group.